test/plugins/features/movie/controller.test.js

```diff
@@ -13,12 +13,47 @@ describe('movie controller', () => {
       return Controller.create(payload)
       .then((movie) => {
         expect(movie.get('title')).to.eql(payload.title);
+      });
+    });
 
-        return new Movie({ id: movie.id }).fetch();
-      })
-      .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
-        expect(movie.get('release_year')).to.eql(payload.release_year);
+  });
+
+  describe('list', () => {
+
+    const movie_one = { title: 'foo', release_year: 2016 };
+    const movie_two = { title: 'bar', release_year: 2016 };
+    const movie_three = { title: 'foo', release_year: 2015 };
```

Comment:
In Javascript, it's super confusing sometimes as to whether you should use snake_case or camelCase since we like snake_case for our API responses but Javascript customs dictate camelCase. Unless we're naming a property of an object that will eventually be returned out in a response, we stick to the JS way and use camelCase with our variables. That way, it's less glaring when we deal with JS built-ins like `.forEach` or `.toString()`.